encoding/_yaml/type/binary.ts

```diff
@@ -117,6 +117,11 @@ function representYamlBinary(object: Uint8Array): string {
 }
 
 function isBinary(obj: Any): obj is Buffer {
+  if (
+    typeof obj !== "object" || !obj || typeof obj.readFromSync !== "function"
```

Comment:
I hate to do this after already leaving a few comments, but couldn't we simplify all of this down to `typeof obj?.readFromSync !== "function"`

Reply:
I think we could. That choice _was_ intentional, though :) I wasn't sure what the preferred style of the repo was. Happy to make the change.

Reply:
Wait now, I think I read `readSync` correctly earlier. We're checking for the downstream read inside `buf.readFromSync`.

Reply:
The check is there because we create a buffer `buf`, and then call

```javascript
buf.readFromSync(obj as Buffer)
```

So we need to check for the part of `obj`'s interface that is correct. That is `readSync`, _not_ `readFromSync`, from [here specifically](https://github.com/denoland/deno_std/blob/main/io/buffer.ts#L247)

Reply:
Ah I see. Thanks!